hello-oboe/app/src/main/cpp/OboeSinePlayer.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+
+#ifndef HELLO_OBOE_OBOESINEPLAYER_H
+#define HELLO_OBOE_OBOESINEPLAYER_H
+
+
+#include <math.h>
+#include <algorithm>
+
+#include <oboe/Oboe.h>
+
```

Comment:
Consider adding a comment describing what this class does. Example: 

This class is responsible for creating an audio stream and starting it. It specifies a callback function `onAudioReady` which is called each time the audio stream needs more data. Inside this callback either silence is rendered or if the `isOn` variable is true a sine wave will be rendered. The sine wave's frequency is hardcoded to 440Hz inside `kFrequency`.